packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -79,13 +72,41 @@ const WorkflowStepStore = types
       addDisposer(self, workflowDisposer)
     }
 
+    function createAnnotationObserver () {
+      const annotationDisposer = autorun(() => {
+        if (self.active?.isThereBranching) {
+          // presumes one single choice task per step
+          const [singleChoiceTask] = self.activeStepTasks.filter(task => task.type === 'single')
+          onAction(getRoot(self), (call) => {
+            if (call.path.endsWith(singleChoiceTask.annotation?.id) && call.name === 'update') {
+              let nextStepKey
+              const singleChoiceTaskAnswers = toJS(singleChoiceTask.answers)
```

Comment:
It's unclear why the conversion here is necessary. I checked and directly using `singleChoiceTask.answers[call.args[0]].next` is working for me. Is there a case when this wouldn't?